src/pages/login/component/form/index.tsx

```diff
@@ -0,0 +1,90 @@
+import { useEffect, useState } from "react";
+import Button from "../../../../components/button";
+import Input from "../../../../components/input";
+import FormError from "../../../../components/form-error";
+
+type errorType = {
+  email: string;
+  password: string;
+};
+
+const LoginForm = () => {
+  const [email, setEmail] = useState<string>("");
+  const [password, setPassword] = useState<string>("");
+
+  const [error, setError] = useState<errorType>({
+    password: "Informe um email ou número de telefone válido.",
+    email: "A senha deve ter entre 4 e 60 caracteres.",
+  });
```

Comment:
acredito que as mensagens estão invertidas.
```suggestion
  const [error, setError] = useState<errorType>({
    password: "A senha deve ter entre 4 e 60 caracteres.",
    email: "Informe um email ou número de telefone válido.",
  });
```